js/iframeResizer.js

```diff
@@ -216,7 +216,7 @@
         log(iframeId, 'Set ' + dimension + ' to max value')
       }
 
-      messageData[dimension] = '' + size
+      messageData[dimension] = Number(size)
```

Comment:
Technically this should be unnecessary, but since an explicit cast was already happening, I kept this symmetrical.

Reply:
I'm just wondering why I originally case this to a string, and if changing this would be a breaking change for anyone.

Reply:
It certainly qualifies as a breaking change.
This was the concern I brought up in https://github.com/davidjbradshaw/iframe-resizer-react/issues/70

It seemed better to introduce this as a breaking change, and to keep the types sensible, rather than adjusting the types to accommodate what felt like an *unusual* API.

Let me know which way you'd prefer moving forward and I can accommodate that direction.

Reply:
Hi Sorry, I've been very ill for the last two years. I would rather avoid breaking changes.

Reply:
My sympathies for your misfortune.
I appreciate that you've revisited this thread. That takes effort and intention.

Feel free to close this PR if it doesn't match the direction you'd like to take this library.

Reply:
Thanks